content/docs/concepts/tsdb.md

```diff
@@ -0,0 +1,52 @@
+---
+title: TSDB
+sort_rank: 5
+---
+
+A times series is a series of **samples**. Each sample is a two-tuple consisting of a timestamp and a value:
```

Comment:
This duplicates docs in our Data Model section.

Reply:
I’m not sure we need to be *that* vigilant about redundancy

Reply:
This is beginner document that'd duplicate beginner documentation that it'd lives right beside.